care/emr/reports/context_builder/data_points/invoice.py

```
@@ -0,0 +1,72 @@
+}
+
+
+class InvoiceReportFilter(filters.FilterSet):
+    status = filters.CharFilter(lookup_expr="iexact")
+    title = filters.CharFilter(lookup_expr="icontains")
+    number = filters.CharFilter(lookup_expr="icontains")
+
+
+class InvoiceContextBuilder(QuerysetContextBuilder):
+    filterset_class = InvoiceReportFilter
+    __filterset_backends__ = [filters.DjangoFilterBackend]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)